src/openrct2-ui/drawing/engines/DrawingEngineFactory.hpp

```diff
@@ -33,14 +33,14 @@ namespace OpenRCT2
             std::unique_ptr<IDrawingEngine> Create(
                 DRAWING_ENGINE_TYPE type, const std::shared_ptr<IUiContext>& uiContext) override
```

Comment:
The type of the parameter should be changed to DrawingEngine, I don't understand how this managed to compile now that the old type is gone.

Reply:
Just realized, this is another type declared IDrawingEngine.h, can you also remove that one and use the correct type?

Reply:
Yes, is another type! I found that strange too haha.
But since I'm new to the code I thought that changing it could affect the functionality.

So you suggest replacing DRAWING_ENGINE_TYPE with the new DrawingEngine enum class, right?
Sure, I'll enhance it.

Reply:
Yeah its an oddity, it should use the right type.